test-harness/scenarios/duplicated-entry-in-enum.oas.scenario

```diff
@@ -0,0 +1,33 @@
+====test====
+Identify enum values that does not respect the specified type
```

Comment:
```suggestion
Identify non-unique enum values
```

Reply:
Ah, oops. Missed it. Going to keep that in mind and will include your suggestion in some other PR.